change/@fluentui-react-color-picker-preview-0f7e3b1e-353f-410a-8642-202d0d30d244.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "fix(react-color-picker): contrast border of thumb",
```

the scope is not needed - its already reflected by the change file

```suggestion
  "comment": "fix: contrast border of thumb",
```